src/fidesapi/main.py

```diff
@@ -169,25 +169,20 @@ def read_index() -> Response:
     return FileResponse(WEBAPP_INDEX)
 
 
-@app.get("/{catchall:path}", response_class=FileResponse, tags=["Default"])
+@app.get(
+    str(WEBAPP_DIRECTORY) + "/{catchall:path}",
```

Comment:
@allisonking Does this satisfy your requirement for a front-end "catch-all", or is there a better way to specify this path?